app/views/partials/inputDropdown.scala.html

```diff
@@ -1,34 +1,53 @@
 @* dropdownOptions as Tuple until data model clearer *@
 
-@(field: Field,
+@(fieldId: String,
 dropdownOptions: Seq[(String, String)],
- args: (Symbol, Any)*
+selectedDropdownOption: Seq[(String, String)],
+attributes: Map[Symbol, Any],
+fieldErrors: Seq[Any]
 )(implicit messages: Messages)
 
 @import views.html.helper._
```

Comment:
It looks like this import is unused. You didn't add it but would you mind getting rid, cheers.